ramls/login.raml

```diff
@@ -128,9 +128,12 @@ traits:
           type: loginCredentials
       responses:
         201:
-          # NOTE: Set-Cookie headers are set in LoginAPI.java using the raw response object
-          # rather than through RMB code-generation. This is because RMB code generation doesn't
-          # let us do multiple headers with the same name, which we need to do here.
+          headers:
+            Set-Cookie:
+              description: access token cookie and refresh token cookie
+              example: |
+                folioAccessToken=abc.def.ghi; Max-Age=600; Expires=Tue, 23 Jan 2024 15:15:49 GMT; Path=/; Secure; HTTPOnly; SameSite=Strict
+                folioRefreshToken=rst.uvw.xyz; Max-Age=604800; Expires=Tue, 30 Jan 2024 15:05:49 GMT; Path=/authn; Secure; HTTPOnly; SameSite=Strict
```

Comment:
The example looks like both cookies are send in a single `Set-Cookie` header.
> To send multiple cookies, multiple Set-Cookie headers should be sent in the same response.

See [Set-Cookie](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie).